task-list/app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,50 @@
+
+
+<header>
+  <h1> To Do List </h1>
+</header>
+
+<nav>
+    <ul>
+      <li><a href='/'>Home</a></li>
+
+      <li><%= link_to "Add a New Task!", tasks_new_path %></li>
+
+      <li><%= link_to "Peeps", people_path %></li>
+    </ul>
+</nav>
+
+
+<main>
+  <!-- <%= params %> -->
```

Comment:
Is this a commented out line for debugging? If so, you should remove before submitting a PR.